### PR TITLE
Unify storages for different printer models

### PR DESCRIPTION
**Pull request by Lowentwickler**
This PR will:
* Fix M1 icons
* Add USB icon for USB storage
* Avoid unnecessary requests to BE for file metadata
* Hardcode storages list to printers

### CODE REVIEW

File: templates/components/preview.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
   ></p>
   <div class="preview-wrapper">
     <div class="preview-img">
-      <img id="preview-img" src="../assets/thumbnail.svg" />
+      <img id="preview-img" src="../assets/thumbnail.svg'" />
```
Comment (ondratu) [inline templates/components/preview.html:80]: I'm not sure about this `'` character in path ...

Comment (Lowentwickler) [reply templates/components/preview.html:80]: Good catch

File: src/printer/components/storage.js [hunk 2]
```diff
@@ -37,12 +31,14 @@ const load = (context, origins, selectedOrigin, onSelect) => {
...
 
     const available = origins.includes(origin);
+    console.log("available", available)
     setVisible(li, available);
     if (!available)
       return;
 
     const location = li.getAttribute("data-location");
     const storageInfo = context.printer.storage?.[location];
+    console.log("selectedOrigin", selectedOrigin)
```
[the remark this answers is not in the document]
Comment (Lowentwickler) [reply src/printer/components/storage.js]: We have some debug outputs in code, due to some weird reports, but definitely not these. Removed.

**Review by filipkotoucek — CHANGES_REQUESTED** [review]

File: config.m1.js [hunk 3]
```diff
@@ -4,7 +4,10 @@ module.exports = (env, args) => {
     const config = {
         PRINTER_NAME: "Original Prusa M1",
         PRINTER_TYPE: "sla",
+        PROJECT_EXTENSIONS: [".sl1", ".sl1s"],
```
[the remark this answers is not in the document]
Comment (Lowentwickler) [reply config.m1.js:7]: This is handled by endpoint. This code is just for compatibility.

File: src/assets/m1/arrow_left.svg [hunk 4]
```diff
@@ -1 +1 @@
-../arrow_left.svg
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" fill-rule="evenodd" clip-rule="evenodd" image-rendering="optimizeQuality" shape-rendering="geometricPrecision" text-rendering="geometricPrecision" viewBox="0 0 363.64 363.64"><path fill="#fff" d="m252.6 0 42.84 42.84-141.47 139.03 141.47 138.94-42.84 42.83L68.2 181.87z"/></svg>
```
[the remark this answers is not in the document]
Comment (Lowentwickler) [reply src/assets/m1/arrow_left.svg:1]: I have a complications with the script that generates icons for M1, so I decided to copy them for now. Some additional efforts are necessary to generate unix symlinks on non-unix OS.

[the remark this answers is not in the document]
Comment (Lowentwickler) [reply src/assets/m1/arrow_left.svg]: I've used `lnk` library to generate them. Should work

File: config.sl1.js [hunk 5]
```diff
@@ -5,7 +5,9 @@ module.exports = (env, args) => {
         PRINTER_NAME: "Original Prusa SL1/S",
         PRINTER_TYPE: "sla",
         PROJECT_EXTENSIONS: [".sl1", ".sl1s"],
```
[the remark this answers is not in the document]
Comment (Lowentwickler) [reply config.sl1.js]: I've changed this in config. But the values are being overwritten with data  from printer

**Review by filipkotoucek — APPROVED** [review]

**Review by ondratu — APPROVED** [review]

**Pull request merged**